README.md

```diff
@@ -138,12 +138,16 @@ after :each do
 end
 ```
 
+## Sequel ##
+If you want to have shared database connections with sequel just
+add the option `single_threaded: true` to your sequel connection in test.
```

Comment:
Would it be possible to set this automatically for people? It seems safe to assume that they want it single threaded any time this gem is active.